controllers/multipleInstancesEnabled/multipleInstancesEnabled.go

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright 2021 IBM Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package multipleinstancesenabled
+
+const MultipleInstancesEnabledTemplate = `
+- name: ibm-management-ingress-operator
+  spec:
+    managementIngress:
+      multipleInstancesEnabled: placeholder
+- name: ibm-ingress-nginx-operator
+  spec:
+    nginxIngress:
+      multipleInstancesEnabled: placeholder
```

Comment:
We don't need to set it for `ibm-ingress-nginx-operator`

Reply:
removed